dashboard/app/models/course.rb

```diff
@@ -51,7 +51,7 @@ def self.load_from_path(path)
     hash = JSON.parse(serialization)
     course = Course.find_or_create_by!(name: hash['name'])
     course.update_scripts(hash['script_names'])
-    course.update!(properties: hash['properties'])
+    course.update!(teacher_resources: hash.try(:[], 'properties').try(:[], 'teacher_resources'))
```

Comment:
https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/course.rb#L34
Not currently. I have some concern that this introduces brittleness for if/when we add new properties. Not sure if there's a better approach.